src/include/sof/schedule/ll_schedule_domain.h

```diff
@@ -46,6 +46,7 @@ struct ll_schedule_domain {
 	spinlock_t lock;		/**< standard lock */
 	atomic_t total_num_tasks;	/**< total number of registered tasks */
 	atomic_t num_clients;		/**< number of registered cores */
+	atomic_t enabled_clients;	/**< number of enabled cores */
```

Comment:
This is confusing - clients should be renamed to cores.

Reply:
> This is confusing - clients should be renamed to cores.

yes, let me change them.